extensions/wikia/AdEngine/js/AdContext.js

```diff
@@ -81,6 +81,11 @@ define('ext.wikia.adEngine.adContext', [
 			isProperCountry(instantGlobals.wgAdDriverHighImpactSlotCountries)
 		) ||  parseInt(qs.getVal('highimpactslot', '0'));
 
+		context.opts.enableScrollHandler = !!(
+			isProperCountry(instantGlobals.wgAdDriverScrollHandlerCountries) ||
+			parseInt(qs.getVal('scrollhandler', '0'))
```

Comment:
Once we merge https://github.com/Wikia/app/pull/8022 we most likely want to re-use `isUrlParamSet()`.